hubblestack/daemon.py

```diff
@@ -214,6 +215,22 @@ def schedule():
             log.error('Scheduled job {0} has an invalid value for seconds or '
                       'splay.'.format(jobname))
         args = jobdata.get('args', [])
+        if 'randomizeSplay' in jobdata and jobdata['randomizeSplay']:
```

Comment:
an example would be useful here. Please add a sample config around line 147 of this file ( in the schedule part of the example config) .

Reply:
Done